lib/ts/styles/styles.css

```diff
@@ -2,19 +2,19 @@
     --palette-background: 255, 255, 255;
     --palette-inputBackground: 250, 250, 250;
```

Comment:
Please mention these changes specifically in the changelog

Reply:
Done